server.js

```diff
@@ -1,17 +1,29 @@
-const bodyParser = require("body-parser");
 const express = require("express");
 const app = express();
 const moment = require("moment");
-moment().format();
 
-app.use(bodyParser.urlencoded({ extended: false }));
-app.use(bodyParser.json());
 app.use(express.static("public"));
 
-app.post("/", function(req, res) {
-  //add request
+app.get("/:query", function(req, res) {
+  const urlString = moment.unix(req.params.query);
+  const validate = urlString.isValid();
+  const naturalDate = moment(urlString).format("MMMM Do YYYY");
+  const unixDate = moment(req.params.query, "MMMM D, YYYY").format("X");
+
+  if (validate) {
+    res.json({
+      unix: req.params.query,
+      natural: naturalDate
+    });
+  } else {
+    res.json({
+      unix: unixDate,
+      natural: req.params.query
+    });
+  }
+  console.log(validate);
```

Comment:
Console logging is good for testing but might be good to remove this now.

Reply:
Removed console.log